subsys/usb/class/cdc_acm.c

```diff
@@ -416,29 +418,38 @@ static void cdc_acm_do_cb(struct cdc_acm_dev_data_t *dev_data,
 		LOG_DBG("Device connected");
 		break;
 	case USB_DC_CONFIGURED:
-		cdc_acm_read_cb(cfg->endpoint[ACM_OUT_EP_IDX].ep_addr, 0,
-				dev_data);
+		LOG_INF("Device configured");
+		if (!dev_data->configured) {
+			LOG_ERR("create transfer on ep 0x%02x",
```

Comment:
I guess it was supposed to be `LOG_INF`.

Reply:
I think `LOG_DBG` would be even better.